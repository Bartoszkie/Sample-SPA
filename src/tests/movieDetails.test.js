import { shallow } from "enzyme";
import React from "react";
import MovieDetails from ".././components/movie-details/movie-details.component";

it("MovieDetails snapshot test", () => {
  expect(shallow(<MovieDetails />)).toMatchSnapshot();
});
