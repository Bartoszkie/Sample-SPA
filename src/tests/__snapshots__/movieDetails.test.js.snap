// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetails snapshot test 1`] = `ShallowWrapper {}`;
